engine/engine/src/engine.cpp

```diff
@@ -1755,6 +1754,7 @@ namespace dmEngine
 
         for (uint32_t i = 0; i < num_steps; ++i)
         {
+            DM_PROFILE("Step");
             // We currently cannot separate the update from the render,
             // since some of the update is done in the render updates (e.g. sprite transforms)
```

Comment:
When Update Frequency isn’t 0 we don’t do a step. We just calculate time step calling `CalcTimeStep` (see 1754) and `num_steps` depends on it. If `num_steps` is 0 - we don’t really do a step.